metadata-ingestion/source_docs/snowflake.md

```diff
@@ -74,20 +76,28 @@ For general pointers on writing and running a recipe, see our [main recipe guide
 source:
   type: snowflake
   config:
+
+    setup: # Optional
+      enabled: false
+      dry_run: true
+      skip_ingestion: true
+      admin_username: "${SNOWFLAKE_ADMIN_USER}"
```

Comment:
what is this admin username for? 

and what does skip ingestion mean? just print?

Reply:
changing to `provision_role` should make this clear now hopefully